ray-operator/test/sampleyaml/support.go

```diff
@@ -57,6 +57,11 @@ func SubmitJobsToAllPods(t Test, rayCluster *rayv1.RayCluster) func(Gomega) {
 		}
 		for _, pod := range pods {
 			for _, container := range pod.Spec.Containers {
+
```

Comment:
We don't need to iterate through all containers. We can directly use the first application container. In KubeRay, the Ray container must always be the first container.
 https://github.com/ray-project/kuberay/blob/aeba37e58813e0e415cb937fb0c977f2cfb981c4/ray-operator/controllers/ray/utils/constant.go#L31.